frontend/src/queries/Query/Query.tsx

```diff
@@ -175,8 +175,8 @@ export function Query<Q extends Node>(props: QueryProps<Q>): JSX.Element | null
         component = <WebVitalsPathBreakdown query={query} cachedResults={props.cachedResults} context={queryContext} />
     } else if (isHogQuery(query)) {
         component = <HogDebug query={query} setQuery={setQuery as (query: any) => void} queryKey={String(uniqueKey)} />
-    } else if (isWebActiveHoursHeatMapQuery(query)) {
-        component = <EventsHeatMap query={query} context={queryContext} cachedResults={props.cachedResults} />
+    } else if (isHeatMapQuery(query)) {
```

Comment:
I would suggest to keep `ActiveHoursHeatmap` or `EventsHeatMapQuery` because we do have heatmaps as well: https://posthog.com/docs/toolbar/heatmaps